frontend/server/src/Controllers/Problem.php

```diff
@@ -3870,13 +3909,16 @@ public static function getProblemEditDetailsForSmarty(
             $r,
             /*$isRequired=*/ false
         );
-
         if (!isset($r['request'])) {
+            print_r($r['request']);
             return [
                 'IS_UPDATE' => true,
                 'LOAD_MATHJAX' => true,
                 'LOAD_PAGEDOWN' => true,
                 'STATUS_SUCCESS' => '',
+                'payload' =>
+                    self::getCommonPayloadForSmarty()
+                ,
```

Comment:
nit:

```suggestion
                'payload' => self::getCommonPayloadForSmarty(),
```

igual abajo